modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/brainparcellationmesh/v0.1.0.json

```diff
@@ -0,0 +1,72 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/brainparcellationmesh/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
+    "{{base}}/schemas/nexus/core/distribution/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:BrainParcellationMeshShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for describing brain parcellation mesh.",
+      "targetClass": "nsg:BrainParcellationMesh",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "schema:distribution",
+              "name": "Distribution",
+              "description": "Brain parcellation mesh location.",
+              "node": "this:BrainParcellationMeshDistributionShape",
+            },
+            {
+              "path": "nsg:brainLocation",
+              "name": "Soma brain location",
+              "description": "Brain Location (brain region, position In Layer, longitudinalAxis) of the reconstructed cell soma.",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/brainlocation/v0.1.0/shapes/BrainLocationShape",
```

Comment:
A used shape should be imported. I would suggest to use the latest neurosciencegraph/commons/brainlocation/v0.1.1/shapes/BrainLocationShape.